keps/sig-node/3085-pod-conditions-for-starting-completition-of-sandbox-creation/README.md

```diff
@@ -1073,6 +1092,7 @@ in back-to-back releases.
 
 #### Beta
 
+- Condition is moved from a package constant in Kubelet to a [API Defined Condition](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L2834)
```

Comment:
are e2e tests part of this milestone or they already implemented? If implemented - please update the section above to point to them. If part of this milestone, can you please make the item `Add more test cases and link to this KEP.` more explicit in this graduation criteria saying that e2e tests will be implemented

Reply:
Sure.  I see these tests https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/pod_conditions_test.go which makes me thing they have been implemented.  

I am not sure how to confirm that they are actually running in the CI.

Reply:
I see you have updated test cases section. Is it referring to tests from this file?

Reply:
So I see the original author implemented a few of the test cases but they mentioned (graceful shutdown and I added a new case for volumes/secrets) so those should be added in beta.